### ISSUE TITLE
Getting blank mask while testing pre-trained model with custom image

### ISSUE THREAD

**Issue by SouravSarkar865** [wgcban/ChangeFormer#69]
Hi.
I tried to use the pretrained model on two custom images and the mask I got was blank despite there being change between the images.I am sharing the before image, after image where I marked the changed point with blue boundary for u to easily figure and also the resulting mask.

![result_mask](https://github.com/wgcban/ChangeFormer/assets/57471303/a1daa23a-cc46-4bc0-89b5-88675fac30a2)
![before](https://github.com/wgcban/ChangeFormer/assets/57471303/d8adf375-2ce6-4ce2-b10b-e107736f9019)
![after](https://github.com/wgcban/ChangeFormer/assets/57471303/a35189ec-1102-41e1-8c56-64628f684abe)

**Comment by wgcban** [comment]
@SouravSarkar865 thanks for sharing this. 

1. First: Did you try giving the example images provided in this repo just to make sure that everything else is going fine?
2. Second: Can you pls try with reshaping/ cropping your images into 256x256 resolution? Because the original model is trained with 256x256 resolution?
3. How the generated mask has a different shape than the input images?

Thanks.

**Comment by SouravSarkar865** [comment]
Hi! 
Thank you so much for your response. I did try the example images from the repo first which were working fine and giving the correct mask. As per your instruction I did reshape my images into 256x256 to make sure the generated mask and input images have the same size, yet I was getting a blank image with no mask.
Is there anything else I should do?

**Comment by SouravSarkar865** [comment]
Thank you for your response.
The images I provided are containing boats and ships so it can be that due to my input images being different from the kind of images the model has been trained with, I am not getting the desired output.

**Comment by wgcban** [comment]
@SouravSarkar865 I see. Yes the assumption is correct. The provided model only detects the changes in buildings and not the changes in boats. So, the model output is correct. If you would like to detect the changes in boats, I would recommend you to fine tune the provided model for boat change detection.
thanks.
